src/certificates/certificates-sidebar/sidebar-block/SidebarBlock.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import SidebarBlock from '.';
+
+describe('SidebarBlock', () => {
+  test('renders without crashing', () => {
+    render(<SidebarBlock title="Test Title" paragraphs={['Test Paragraph']} isLast={false} />);
```

Comment:
Similar for [this](https://github.com/raccoongang/frontend-app-course-authoring/pull/119/files#diff-af05d2d285765c1750ddc562ea91ef818a4f3a35f18738925d71c5cab5c67369R21)

Reply:
i did it because [this](https://github.com/raccoongang/frontend-app-course-authoring/pull/119/files#diff-af05d2d285765c1750ddc562ea91ef818a4f3a35f18738925d71c5cab5c67369R21) component needed to be wrapped with provider etc, but SidebarBlock is a simple dummy component